examples/HelloSO3.cpp

```diff
@@ -0,0 +1,40 @@
+#include "sophus/geometry.hpp"
+#include <iostream>
+
+int main() {
+
+    // The following demonstrates the group multiplication of rotation matrices
+
+    // Create rotation matrices from rotations around the x and y and z axes:
+    Sophus::SO3d R1 = Sophus::SO3d::rotX(M_PI/4);
```

Comment:
Please use 
``const double kPi = Sophus::Constants<double>::pi;``
Strictly speaking ``M_PI`` is not part of the c++ standard.